coreid-nextjs-typescript/server/business/assently/assentlyUserFlow.ts

```diff
@@ -0,0 +1,71 @@
+import jwt, { JwtPayload } from "jsonwebtoken"
+import uuid4 from "uuid4"
+
+export class AssentlyTokenUserFlow {
+    private static secret = process.env.ASSENTLY_SECRET! // secret provided by assently, used to sign request token
+    private static identityKey = process.env.ASSENTLY_IDENTITY_KEY! // secret identity key by assently, used to validate response
+    private static issuer = process.env.ASSENTLY_ISSUER! // issuer provided by assently usually your company or name
+    private static domain_name = process.env.ASSENTLY_DOMAIN_NAME! // your personal identifier, during testing use "Test av Mobilt BankID"
```

Comment:
With the previous change commented we of course would need to update usage of it as well. 
To reduce changes needed I decided to leave the variable name as is.
```suggestion
    private static domain_name = process.env.ASSENTLY_CUSTOM_DISPLAY_NAME! // your personal identifier, during testing use "Test av Mobilt BankID"
```